### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by drahnr** [rpm-rs/rpm#81] [General]
I just uploaded a first draft to the org, I'd like some feedback. Any feedback is welcome - I will upload the source after the first iteration of fixes.

**Comment by dralley** [comment]
I'm the furthest thing from a graphical design inclined person so I have no useful feedback, it looks pretty good to me!

**Reply by dralley** [reply]
On second thought, even though I like the design, it might be too similar to the original to use without permission?
